gadget/install/install.go

```diff
@@ -138,7 +141,9 @@ func Run(model gadget.Model, gadgetRoot, kernelRoot, device string, options Opti
 		logger.Noticef("created new partition %v for structure %v (size %v) %s",
 			part.Node, part, part.Size.IECString(), roleFmt)
 		if options.Encrypt && roleNeedsEncryption(part.Role) {
+			sp = perfTimings.StartSpan("make-key-set", fmt.Sprintf("Create encryption key set for %s", part.Node))
```

Comment:
you probably know but there are more time consuming things done here for the encrypted case